pettingzoo/sisl/multiwalker/multiwalker.py

```diff
@@ -160,6 +161,14 @@ def __init__(self, *args, **kwargs):
         # spaces
         self.action_spaces = dict(zip(self.agents, self.env.action_space))
         self.observation_spaces = dict(zip(self.agents, self.env.observation_space))
+        self.state_space = spaces.Box(
```

Comment:
My only question is should this code be under multi walker base? The other two spaces are, but maybe it’s not possible